python/cudf/dataframe/dataframe.py

```diff
@@ -2042,6 +2042,112 @@ def fillna(self, value, method=None, axis=None, inplace=False, limit=None):
         if not inplace:
             return outdf
 
+    def describe(self, percentiles=None, include=None):
```

Comment:
Even if not yet supported we should add the `exclude` parameter and throw an exception if anything other than default value of `None` is passed. This will help with Dask compatibility I'm sure.

Reply:
Good point. This should be at the series level as well for compatibility.

Reply:
Updated.